scripts/cif_validate: tests/cases/cif_validate_058.inp data_proper_local_names: NOTE, definition of the '_category.[local]_new_item' data item was not found in the provided dictionaries.
scripts/cif_validate: tests/cases/cif_validate_058.inp data_proper_local_names: NOTE, definition of the '_category.next_item' data item was not found in the provided dictionaries.
scripts/cif_validate: tests/cases/cif_validate_058.inp data_incorrect_local_names: NOTE, definition of the '_category._[local]_extra_underscore' data item was not found in the provided dictionaries.
scripts/cif_validate: tests/cases/cif_validate_058.inp data_incorrect_local_names: NOTE, definition of the '_category.incorrect_[local]_placement' data item was not found in the provided dictionaries.
scripts/cif_validate: tests/cases/cif_validate_058.inp data_incorrect_local_names: NOTE, definition of the '_local_category.rest_item' data item was not found in the provided dictionaries.
